assets/csidriveroperators/aws-ebs/09_deployment.yaml

```diff
@@ -11,6 +11,8 @@ spec:
   strategy: {}
   template:
     metadata:
+      annotations:
+        workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
```

Comment:
Just a heads up: there's a PR in the merge queue that adds Azure Disk operator, so we need to patch that deployment too: https://github.com/openshift/cluster-storage-operator/pull/148

Reply:
Thanks for the comment, I asked under the PR to add it.